extensions/wikia/Search/classes/Services/EntitySearchService.php

```diff
@@ -7,7 +7,7 @@
 use Wikia\Search\QueryService\Factory;
 use WikiFactory;
 
-class EntitySearchService {
+class EntitySearchService extends AbstractSearchService {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "EntitySearchService" has 35 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)